nilearn/datasets/atlas.py

```diff
@@ -889,10 +975,25 @@ def fetch_atlas_aal(version='SPM12', data_dir=None, url=None, resume=True,
     see :footcite:`AAL_atlas`,
     and :footcite:`TZOURIOMAZOYER2002273`.
 
+    .. warning::
+
+        The maps image (``data.maps``) contains 117 unique integer values
+        defining the parcellation. However, these values are not consecutive
+        integers from 0 to 116 as is usually the case in Nilearn.
+        Therefore, these values shouldn't be interpreted as indices for the
+        list of label names.
+        For example, to get the name of the region corresponding to label
+        5021 in the image, you should do:
+
+        .. code-block::python
```

Comment:
```suggestion
        .. code-block:: python
```

The code block doesn't show up (see below).

<img width="879" alt="Screen Shot 2021-12-13 at 11 40 07 AM" src="https://user-images.githubusercontent.com/8228902/145852624-fd42e1da-80c5-4cd3-b6d0-fef8939f11d3.png">

Reply:
Thanks! :+1: